solr/core/src/java/org/apache/solr/highlight/UnifiedSolrHighlighter.java

```diff
@@ -314,6 +343,24 @@ protected PassageFormatter getFormatter(String fieldName) {
       return new DefaultPassageFormatter(preTag, postTag, ellipsis, "html".equals(encoder));
     }
 
+    @Override
+    protected Comparator<Passage> getPassageSortComparator(String fieldName) {
+      PassageSort passageSort =
+          PassageSort.parse(
+              params.getFieldParam(fieldName, HighlightParams.PASSAGE_SORT, "startOffset"));
```

Comment:
could you instead reference the START_OFFSET.key instead of this constant string?

Reply:
Thank you for the review. The feedback you provided has been addressed in commit 210a5ec.